ts/components/session/icon/SessionIconButton.tsx

```diff
@@ -3,8 +3,18 @@ import classNames from 'classnames';
 
 import { Props, SessionIcon } from '../icon';
 
-export class SessionIconButton extends React.PureComponent<Props> {
-  public static defaultProps = SessionIcon.defaultProps;
+interface SProps extends Props {
+  onClick: any;
+}
+
+export class SessionIconButton extends React.PureComponent<SProps> {
+  public static readonly extendedDefaults = {
+    onClick: () => null,
+  };
+  public static readonlydefaultProps = {
```

Comment:
missing a space between `readonly` and `defaultProps`

Reply:
Fixed.